subsys/bluetooth/host/settings.c

```diff
@@ -318,6 +318,10 @@ int bt_settings_store(const char *key, uint8_t id, const bt_addr_le_t *addr, con
 		}
 	}
 
+#if defined(CONFIG_BT_TESTING)
+	bt_testing_settings_store_hook(key_str, value, val_len);
+#endif
```

Comment:
Can this use `IF_ENABLED()`?

Reply:
We will get "implicit declaration of function" errors in applications that uses settings but without `CONFIG_BT_TESTING` :/